packages/ts-client/src/FishjamClient.ts

```diff
@@ -714,6 +726,29 @@ export class FishjamClient<
     });
   }
 
+  private sendStatistics() {
+    this.getStatistics(null).then((statistics) => {
+      const tracksStatistics: Record<
+        string,
+        RTCInboundRtpStreamStats | RTCOutboundRtpStreamStats
+      > = {};
+
+      statistics.forEach((report, key) => {
+        if (report.type === 'inbound-rtp' || report.type === 'outbound-rtp')
+          tracksStatistics[key] = report;
+      });
+
+      const mediaEvent = generateMediaEvent('RTCStatsReport', tracksStatistics);
+      const serializedMediaEvent = serializeMediaEvent(mediaEvent);
+
+      const message = PeerMessage.encode({
+        mediaEvent: { data: serializedMediaEvent },
+      }).finish();
+
+      this.websocket?.send(message);
+    });
```

Comment:
Wouldn't it be simpler to just send all types of reports and then filter it on the backend side? Because currently if some user's browser has changed type we won't see it. Also if we would want to gather more statistics e.g: candidate statistics it will require modification in web-client-sdk.